packages/editor/src/store/effects/posts.js

```diff
@@ -313,6 +313,7 @@ export const refreshPost = async ( action, store ) => {
 	const postType = await resolveSelector( 'core', 'getPostType', postTypeSlug );
 	const newPost = await apiFetch( {
 		path: `/wp/v2/${ postType.rest_base }/${ post.id }?context=edit`,
+		usePreloaded: false,
```

Comment:
The only thing that's bothering me is that the `editor` package is now aware of the existence of a potential preloading middleware.

No strong opinions but an alternative could be to add a timestamp to the query string.

Reply:
I was considering using some more or less random qarg. I like the idea of a timestamp.

Reply:
Is there precedent for the timestamp qarg? Just looking for possibly established nomenclature...

Reply:
I'm not aware of any personally

Reply:
Timestamp qarg PR: #12060

Reply:
This does the job, but probably the other PR looks a little better - I like the simplicity.

No matter which one we choose - let's make sure to document this caching feature thoroughly.